### ISSUE TITLE
Implement Normalization Layers

### ISSUE THREAD

**Issue by jessegrabowski** [jessegrabowski/pytensor-ml#11]
These are very standard and important for pretty much all architectures

- [ ] [Batch Norm](https://pytorch.org/docs/stable/generated/torch.nn.BatchNorm1d.html)
- [ ] [Layer Norm](https://pytorch.org/docs/stable/generated/torch.nn.LayerNorm.html)

**Comment by jessegrabowski** [comment]
`BatchNorm2d` was added in #18 . Still need `LayerNorm`.
